frontend/package.json

```diff
@@ -11,7 +11,8 @@
     "test:e2e": "playwright test",
     "test": "bun run test:lint && bun run test:types && bun run test:e2e",
     "test:spelling": "bunx cspell src/**/*.{tsx,ts}",
-    "clean": "rm -rf node_modules dist bun.lockb && bun pm cache rm"
+    "clean": "rm -rf node_modules dist bun.lockb && bun pm cache rm",
+    "test:lighthouse": "playwright test tests/lighthouse.spec.ts"
```

Comment:
I don't think this is really necessary. If we were going to add it, it'd make sense to add another script for Axe as well.

But you can just run e.g. `bun run test:e2e lighthouse`, using [Playwright's path argument](https://playwright.dev/docs/test-cli#reference), which is almost as short and more flexible.